_security-analytics/usage/findings.md

```diff
@@ -7,11 +7,11 @@ nav_order: 35
 
 # Working with findings
 
-The Findings window includes features for viewing and working with findings. The two main features are:
+The **Findings** window includes features for viewing and working with findings. The two main features are:
 * The bar graph with findings information arranged by count, date, and log type or rule severity.
-* The Findings list arranged by time, finding ID, rule name, and other details.
+* The **Findings** list arranged by time, finding ID, rule name, and other details.
 
-You can select the **Refresh** button at any time to refresh information on the Findings page.
+You can select the **Refresh** button at any time to refresh information on the **Findings** page.
```

Comment:
We are now referring to button clicks with the word "Choose" as opposed to "Select" (global comment)

Reply:
We should as a team have a conversation about this. I have changed "select" to "choose" and "choose" to "select" in a number of PRs lately. Different reviewers, different interpretations of the style guide.